docs/src/models/biological.md

```diff
@@ -1,166 +1,307 @@
+```julia
+rn1 = @reaction_network begin
+ (1.0,(1.0,2.0)), S ↔ (P1,P2)  
+end
+rn2 = @reaction_network begin
+  1.0, S → P1
+  1.0, S → P2
+  1.0, P1 → S   
+  2.0, P2 → S 
 end
 ```
 
-By using the `@reaction_func` macro it is possible to define your own functions,
-which may then be used when creating new reaction networks.
+#### Production and Destruction and Stoichiometry
+Sometimes reactants are produced/destroyed from/to nothing. This can be designated using either `0` or `∅`:
+```julia
+rn = @reaction_network begin
+  2.0, 0 → X
+  1.0, X → ∅
+end
+```
+Sometimes several molecules of the same reactant is involved in a reaction, the stoichiometry of a reactant in a reaction can be set using a number. Here two species of `X` forms the dimer `X2`:
+```julia
+rn = @reaction_network begin
+  1.0, 2X → X2
+end
+```
+this corresponds to the differential equation:
+d[X]/dt = -1.0\*[X\]^2/2!
+d[X2]/dt = 1.0\*[X\]^2/2!
+Other numbers than 2 can be used and parenthesises can be used to use the same stoichiometry for several reactants:
+```julia
+rn = @reaction_network begin
+  1.0, X + 2(Y + Z) → XY2Z2
+end
+```
 
+#### Variable reaction rates
+Reaction rates do not need to be constant, but can also depend on the current concentration of the various reactants (when e.g. one reactant activate the production of another one). E.g. this is a valid notation:
+```julia
+rn = @reaction_network begin
+  X, Y → ∅
+end
+```
+and will have `Y` degraded at rate 
+d[Y]/dt = -[X]\*[Y]
+Note that this is actually equivalent to the reaction
+```julia
+rn = @reaction_network begin
+  1.0, X + Y → X
+end
+```
+Most expressions and functions are valid reaction rates, e.g:
 ```julia
-@reaction_func hill2(x, v, k) = v*x^2/(k^2+x^2)    
-rn = @reaction_network rType begin
-  (1.0,hill2(XY,1.5,2.0)), X + Y ↔ XY
+rn = @reaction_network begin
+  2.0*X^2, 0 → X + Y
+  gamma(Y)/5, X → ∅
+  pi*X/Y, Y → ∅
 end
 ```
+please note that user defined functions cannot be used directly (see later section "User defined functions in reaction rates").
 
-Reaction rates are automatically adjusted according mass kinetics, including
-taking special account of higher order terms like `2X -->`. This can be disabled
-using any non-filled arrow (`⇐, ⟽, ⇒, ⟾, ⇔, ⟺`), in which case the reaction
-rate will be exactly as input. E.g the two reactions in
+#### Defining parameters
+Just as when defining normal differential equations using `DifferentialEquations` parameter values does not need to be set when the model is created. Components can be designated as parameters by declaring them at the end:
+```julia
+rn = @reaction_network begin
+  p, ∅ → X
+  d, X → ∅
+end p d
+```
+Parameters can only exist in the reaction rates (where they can be mixed with reactants). All variables not declared at the end will be considered a reactant.
 
+#### Pre-defined functions
```

Comment:
I think you might need a space under the headers for them to parse correctly.

Reply:
Unsure, I checked everything by imputing it to a dummy message in the discourse and it looked fine there, not excluding that you still might be right or I am misunderstanding.